winter_messaging_outbox_transacton/start_worker.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import importlib
+import logging
+import sys
+
+from injector import Injector
+
+from winter_messaging_outbox_transacton.consumer_worker import ConsumerWorker
+from winter_messaging_outbox_transacton.injection import Configuration
+from winter_messaging_outbox_transacton.middleware_registry import MiddlewareRegistry
+from winter_messaging_outbox_transacton.worker_configuration import WorkerConfiguration
+
+logger = logging.getLogger('event_handling')
+
+
+class Parser(argparse.ArgumentParser):
+    def error(self, message):
+        sys.stderr.write('error: %s\n' % message)
+        self.print_help()
+        sys.exit(2)
+
+
+def parse_args(parser_object):
+    parser_object.add_argument(
+        'consumer',
+        type=str,
+        help='Consumer ID',
+    )
+    parser_object.add_argument(
+        '-p',
+        '--package-module',
+        type=str,
+        dest='package_module',
+        help='',
+    )
+    return parser_object.parse_args()
+
+
+if __name__ == '__main__':
+    import django
+    import winter.core
+    import winter_django
+
+    parser = Parser(description='Run consumer worker')
+    args = parse_args(parser)
+
+    consumer_id = args.consumer
+    package_name = args.package_module
+
+    try:
+        worker_module = importlib.import_module(f'{package_name}.worker')
+    except ModuleNotFoundError:
+        worker_module = None
+
+    winter_django.setup()
+    django.setup()
+
+    if worker_module and isinstance(worker_module.worker_configuration, WorkerConfiguration):
```

Comment:
you can remove if worker_module and leave only isinstance (None is not instance WorkerConfiguration)